blockchain/blockchain-wallet-v4-frontend
# feat(KYC): Tiers should display as levels, remove mobile

## Description
2111, 2110

## Change Type
Please enter one or more of the following: 
- Feature

## Testing Steps
Visit tiers pages, check tiers messages in SwapUpgrade modal, check tiers messages on checkout

## Code Checklist
